pedalboard/plugins/PitchShift.h

```diff
@@ -30,48 +30,45 @@ class PitchShift : public RubberbandPlugin
 
 {
 private:
-  double _scaleFactor = 1.0;
+  double _semitones = 0.0;
 
   // Allow pitch shifting by up to 6 octaves up or down:
-  static constexpr float MIN_SCALE_FACTOR = 0.015625;
-  static constexpr float MAX_SCALE_FACTOR = 64.0;
+  static constexpr int MIN_SEMITONES = -6 * 12;
+  static constexpr int MAX_SEMITONES = 6 * 12;
 
 public:
-  void setScaleFactor(double scale) {
-    if (scale < MIN_SCALE_FACTOR || scale > MAX_SCALE_FACTOR) {
-      throw std::range_error("Scale factor must be a value between " +
-                             std::to_string(MIN_SCALE_FACTOR) + " and " +
-                             std::to_string(MAX_SCALE_FACTOR) + ".");
+  void setSemitones(double semitones) {
+    if (semitones < MIN_SEMITONES || semitones > MAX_SEMITONES) {
+      throw std::range_error("Semitones of pitch must be a value between " +
+                             std::to_string(MIN_SEMITONES) + "st and " +
+                             std::to_string(MAX_SEMITONES) + "st.");
     }
 
-    _scaleFactor = scale;
+    _semitones = semitones;
 
     if (rbPtr)
-      rbPtr->setPitchScale(_scaleFactor);
+      rbPtr->setPitchScale(getScaleFactor());
   }
 
-  double getScaleFactor() { return _scaleFactor; }
+  double getSemitones() { return _semitones; }
+  double getScaleFactor() { return pow(2, (getSemitones() / 12)); }
```

Comment:
Do you want to expose an option to retrieve scale factor in the python bindings as well? Otherwise is it worth making this function private / protected?